api/api_test.go

```diff
@@ -760,6 +760,52 @@ func TestListRequestsToJoinTeam(t *testing.T) {
 	})
 }
 
+func TestDeleteRequestToJoinTeam(t *testing.T) {
+	t.Run("parses the name from a good response", func(t *testing.T) {
+		client, mux, _, teardown := setup()
+		defer teardown()
+
+		teamUUID := uuid.Must(uuid.NewV4())
+		requestUUID := uuid.Must(uuid.NewV4())
+
+		mockResponseHandler := func(w http.ResponseWriter, r *http.Request) {
+			assertClientSentVerb(t, "DELETE", r.Method)
+			w.Header().Add("Content-Type", "application/json")
+			w.WriteHeader(http.StatusAccepted)
+		}
+		mux.HandleFunc(
+			fmt.Sprintf("/team/%s/requests-to-join/%s", teamUUID, requestUUID),
+			mockResponseHandler,
+		)
+
+		err := client.DeleteRequestToJoinTeam(teamUUID, requestUUID)
+
+		assert.NoError(t, err)
+	})
+
+	t.Run("passes us error codes", func(t *testing.T) {
+		client, mux, _, teardown := setup()
+		defer teardown()
+
+		teamUUID := uuid.Must(uuid.NewV4())
+		unknownRequestUUID := uuid.Must(uuid.NewV4())
+
+		mockResponseHandler := func(w http.ResponseWriter, r *http.Request) {
+			assertClientSentVerb(t, "DELETE", r.Method)
+			w.Header().Add("Content-Type", "application/json")
```

Comment:
The API doesn't send `application/json` for this endpoint (it doesn't send any body)

Reply:
Ah, yes, good spot!

Reply:
You can self-merge when this is changed (in 2 places: the other test, too)